contribs/src/main/java/com/netflix/conductor/contribs/publisher/TaskNotification.java

```diff
@@ -39,11 +45,30 @@ public TaskNotification(Task task) {
 
     }
 
+    String toJsonStringWithoutInputOutput() {
+        String jsonString;
+        SimpleBeanPropertyFilter theFilter = SimpleBeanPropertyFilter
+                .serializeAllExcept("input","output");
+        FilterProvider provider = new SimpleFilterProvider()
+                .addFilter("SecretRemovalFilter", theFilter);
+        try {
+            jsonString = objectMapper.writer(provider).writeValueAsString(this);
+        } catch (JsonProcessingException e) {
+            LOGGER.error("Failed to convert Task: {} to String. Exception: {}", this, e);
+            throw new RuntimeException(e);
+        }
+        return jsonString;
+    }
+
     String toJsonString() {
```

Comment:
I don't think we should have an available ToJson which includes input/output, someone in the future will include it and it will result in the same problem. We should refactor this function to strip off input/output. Add comments that input/output can have private data